ldms/src/sampler/zfs_common/zfs_common.c

```diff
@@ -0,0 +1,144 @@
+
+/* -*- c-basic-offset: 8 -*- */
+/* Copyright 2022 Lawrence Livermore National Security, LLC
+ * See the top-level COPYING file for details.
+ *
+ * SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
+ */
+
+#define _GNU_SOURCE
+/* Next we include the headers to bring in the zfslib in action */
```

Comment:
```suggestion
/* Next we include the headers to bring in the libzfs in action */
```

Reply:
Very good suggestion. I'll get on it right away.